src/emulator/commandUtils.ts

```diff
@@ -203,6 +205,10 @@ export function setExportOnExitOptions(options: any) {
       // firebase emulators:start --debug --import '' --export-on-exit ''
       throw new FirebaseError(EXPORT_ON_EXIT_USAGE_ERROR);
     }
+
+    if (path.resolve(".").includes(path.resolve(options.exportOnExit))) {
```

Comment:
```suggestion
    if (path.resolve(".").startsWith(path.resolve(options.exportOnExit))) {
```

Prevents false positives like `/a/b/c/d` and `/b/c/d`

Reply:
Nice observations @yuchenshi. I've also included a test for this cenario